.gitignore

```diff
@@ -24,3 +24,5 @@ openqa.log
 t/data/openqa/factory/hdd/00099963-hdd_image2.qcow2
 t/data/openqa/factory/hdd/hdd_image.qcow2
 t/diagram-v*.png
+assets/cache/
+assets/assetpack.db
```

Comment:
You're missing the point, but you will probably experience it soon enough when/if you run into weird problems.

The one thing that might save you is that you're using submodules instead of fetching the assets from web. Not sure.

I wish you good luck, but keep this in mind if things doesn't work out.